packages/i3.rb

```diff
@@ -42,4 +42,16 @@ def self.install
       system "for f in \$(ls #{CREW_BUILD}-*); do g=\$(echo \$f | sed 's,#{CREW_BUILD}-,,'); ln -sf \$f \$g; done"
     end
   end
+  
+  def self.postinstall
+      puts
+      puts "To use this package, you need to download XServer XSDL from Google Play Store".lightblue
+      puts "Use 'starti3' instead of 'i3' to execute this package".lightblue
+      system "touch /usr/local/bin/starti3"
+      system "echo '#!/bin/sh' >> /usr/local/bin/starti3"
+      system "echo 'stopsommelier' >> /usr/local/bin/starti3"
+      system "echo 'export DISPLAY=100.115.92.2:0' >> /usr/local/bin/starti3"
+      system "echo 'i3 & $1 $2 $3 $4' >> /usr/local/bin/starti3"
+      system "chmod +x /usr/local/bin/starti3"
```

Comment:
Can we add the script to the install section?  That way it can be removed when the package is removed and requires less configuration for the user.  Might as well update the package while you're at it also.  https://i3wm.org/downloads/i3-4.18.3.tar.bz2

Reply:
Done